ru-ru/[pyqt-ru] New translation

```diff
@@ -0,0 +1,83 @@
+---
+category: tool
+tool: PyQT
+filename: learnpyqt.py
```

Comment:
Which one exactly? How exactly should I name the file? It's for the future, as I'd like to translate a couple more tutorials.